### PR TITLE
[fix] - Table Generic Hook const

### PR DESCRIPTION
**Pull request by ibelar**
From private to public

### CODE REVIEW

File: src/TableColumn/Generic.php [hunk 1]
```diff
@@ -84,6 +84,7 @@ class Generic
      */
     public function __construct($defaults = [])
     {
+        $t = self::HOOK_GET_HTML_TAGS;
```
Comment (mvorisek) [inline src/TableColumn/Generic.php]: remove?

**Review by mvorisek — APPROVED** [review]
LGTM except some extra staged/committed code

**Review by DarkSide666 — APPROVED** [review]
LGTM, but do not need that line

File: src/TableColumn/Generic.php (hunk 1 above)
Comment (ibelar) [reply src/TableColumn/Generic.php]: oops! done.

**Pull request merged**